src/code-generator/test/non-interactive-cli.test.ts

```diff
@@ -10,56 +10,123 @@ beforeEach(async () => {
 });
 
 afterEach(async () => {
-  await fsExtra.remove(emptyFolderForATest);
+  //await fsExtra.remove(emptyFolderForATest);
 });
 
 describe("Non-interactive CLI component tests", () => {
-  test("When installing with specified app name, then the created folder has this specific name", async () => {
-    // Arrange
-    const specificAppName = "test";
+  describe("ORM type", () => {
+    test("When ORM type is Prisma, then the created DAL folder has prisma dependency and files", async () => {
+      // Arrange
 
-    // Act
-    await execa("ts-node", [
-      "./bin/cli.ts",
-      "immediate",
-      `--target-directory=${emptyFolderForATest}`,
-      `--app-name=${specificAppName}`,
-    ]);
+      // Act
+      await execa("ts-node", [
+        "./bin/cli.ts",
+        "immediate",
+        `--target-directory=${emptyFolderForATest}`,
+        `--app-name=test`,
+        "--orm=prisma",
+      ]);
 
-    // Assert
-    const generatedSolutionFolder = path.join(
-      emptyFolderForATest,
-      specificAppName
-    );
-    const pathExists = await fsExtra.pathExists(generatedSolutionFolder);
-    expect({ pathExists }).toStrictEqual({ pathExists: true });
-    const destinationFolderContent = await fsExtra.readdir(
-      generatedSolutionFolder
-    );
-    expect(destinationFolderContent.length).toBeGreaterThan(0);
-  });
+      // Assert
+      const rootPath = path.join(
+        emptyFolderForATest,
+        "test",
+        "services",
+        "order-service"
+      );
+      const isPrismaInPackageJSON = await testHelpers.doesFileContainPhrase(
+        path.join(rootPath, "package.json"),
+        "prisma"
+      );
+      const isSequelizeInPackageJSON = await testHelpers.doesFileContainPhrase(
+        path.join(rootPath, "package.json"),
+        "sequelize"
+      );
+      const isPrismaFolderInDALLayer = await testHelpers.doesFolderExistInPath(
+        path.join(rootPath, "data-access", "prisma")
+      );
+      expect({
+        isSequelizeInPackageJSON,
+        isPrismaInPackageJSON,
+        isPrismaFolderInDALLayer,
+      }).toStrictEqual({
+        isSequelizeInPackageJSON: false,
+        isPrismaFolderInDALLayer: true,
+        isPrismaInPackageJSON: true,
+      });
+    });
+
+    test("When ORM type is sequelize, then the created DAL folder has only sequelize dependency and files", async () => {
+      console.log(emptyFolderForATest);
+      // Arrange
```

Comment:
If no arrange please remove 😀 (apply for the rest as well)

Reply:
Sure?

I like to keep the structure always predictable